### DISCUSSION TITLE
menu嵌套一个anchor但是无法调菜单项高度导致anchor内容无法正常显示

### DISCUSSION THREAD

**Discussion by ustcstx** [tusen-ai/naive-ui#5895] [Q&A]
const menuOptions: MenuOption[] = [
 {
    label: "历史记录",
    children: [
      {
        label: () => h(
          NAnchor,
          {
            showRail: true,
          },
          myArray.map(item => {
            return h(NAnchorLink, { title: item});
          })
        ),
        key: 'no3'
      },]
<n-menu v-model:value="activeKey" :collapsed="collapsed" :collapsed-width="64" :collapsed-icon-size="22"
          :options="menuOptions" style="line-height: 100px">
        </n-menu>
如以上代码，请帮我解决一下问题

**Comment by jahnli** [comment]
提供最小复现链接
